LanguageFeatures/Control-flow-collections/Null-aware-elements/type_inference_set_A02_t01.dart

```diff
@@ -19,9 +19,8 @@
 ///   - The inferred set element type is `NonNull(U)`. The value added to the
 ///     set will never be `null`.
 ///
-/// @description Check that the inferred type of element `?e1` in the context
-/// `Set<Ps> is `NonNull(U)`, where `U` is the inferred type of the expression
-/// `e1` in the context `Ps?`
+/// @description Check that if the context type of element `?e1` is `Ps`, then
+///  the inferred type of the expression `e1` in the context `Ps?`.
```

Comment:
That's not quite a sentence. Perhaps?:
```suggestion
/// @description Check that if the context type of the element `?e1` is `Ps`
/// then the context type of `e1` is `Ps?`.
```
```suggestion
/// the context type of `e1` is `Ps?`.
```